bin/runtime/src/lib.rs

```diff
@@ -139,6 +141,9 @@ const MAX_BLOCK_WEIGHT: Weight = 400 * WEIGHT_PER_MILLIS;
 // We agreed to 5MB as the block size limit.
 pub const MAX_BLOCK_SIZE: u32 = 5 * 1024 * 1024;
 
+pub const UNITS: Balance = 10_000_000_000;
+pub const DOLLARS: Balance = UNITS; // 10_000_000_000
```

Comment:
There are CENTS defined in this file, so you can reuse them and possibly update them.

Reply:
@mike1729 's comment is addressed and done. @h4nsu I suppose this should be done in a separate PR?

Reply:
Well, it's a small local change (basically renaming constants) that requires bumping the runtime version. I think it's just less hassle to do it together with some other bigger change, but no big deal, up to you.